### PR TITLE
UCS/STUBS: Add stub function which returns 1

### PR DESCRIPTION
**Pull request by yosefe**

### CODE REVIEW

File: src/ucs/sys/stubs.c [hunk 1]
```diff
@@ -22,6 +22,11 @@ int64_t ucs_empty_function_return_zero_int64()
     return 0;
 }
 
+int ucs_empty_function_return_one()
```
Comment (dmitrygx) [inline src/ucs/sys/stubs.c]: Maybe `unsigned` return value is better as we have for `...return_zero`?

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/11413/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 33 workers (click for details)</summary>

**Note:** the logs will be deleted after 28-Jul-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/3bb15c8c8e417023ba4443b41ada3b65/raw/9dd61bffa58a9140d7ef94f17ee5bf4cbeee9554/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/3bb15c8c8e417023ba4443b41ada3b65/raw/a264a5792d898ba9ab2152967b35ed2469325cf6/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/3bb15c8c8e417023ba4443b41ada3b65/raw/5cf9f1ac4e9dbf82d3e9eecefff240c63088b8bd/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/3bb15c8c8e417023ba4443b41ada3b65/raw/d5d8fceb2797e55454d1105506cdf9a143779d91/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/3bb15c8c8e417023ba4443b41ada3b65/raw/7cea099a003f46bf8cfbc4347a155385a5e4dba5/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/3bb15c8c8e417023ba4443b41ada3b65/raw/ba6043b7148c2d42b1427cfe3cd6d1c2da98a870/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/3bb15c8c8e417023ba4443b41ada3b65/raw/80bffa2afa611e0283355b20cbba8a47340bd8fc/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/3bb15c8c8e417023ba4443b41ada3b65/raw/599c316482d045891d0c48715e9f80e86c2534fd/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/3bb15c8c8e417023ba4443b41ada3b65/raw/ec3644d3fdcb07bb0dcd72a404f439e7e596a405/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W0](https://gist.githubusercontent.com/mellanox-github/3bb15c8c8e417023ba4443b41ada3b65/raw/624c62884c3c816e6818f1e36d32ea46515d7617/hpc-test-althca_W0) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W1](https://gist.githubusercontent.com/mellanox-github/3bb15c8c8e417023ba4443b41ada3b65/raw/2e75c97d2e7f3886ef8336c990b86e8d5a6c1393/hpc-test-althca_W1) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W2](https://gist.githubusercontent.com/mellanox-github/3bb15c8c8e417023ba4443b41ada3b65/raw/9452183da2822904220df4707f1a269f6d14c24f/hpc-test-althca_W2) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W3](https://gist.githubusercontent.com/mellanox-github/3bb15c8c8e417023ba4443b41ada3b65/raw/beceb6a34e5e1924365fc4ef6ccf6203318eaa73/hpc-test-althca_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/3bb15c8c8e417023ba4443b41ada3b65/raw/819582a35149d2a25810b3b5b7c1896135fca1a6/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/3bb15c8c8e417023ba4443b41ada3b65/raw/2f2e12c4629d457d692b35b2ecc37b81e29cd8b5/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/3bb15c8c8e417023ba4443b41ada3b65/raw/ed802b40ed72f426b8a8c07145460baacfc7407e/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/3bb15c8c8e417023ba4443b41ada3b65/raw/24144289064c5d26adf3feef8633691af87b0509/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/3bb15c8c8e417023ba4443b41ada3b65/raw/5baeaf0f6d52436bea0a59d45f7b6de51f6172db/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/3bb15c8c8e417023ba4443b41ada3b65/raw/a127b248c3027dbcc0ed63e3cb638dd90cc2477e/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/3bb15c8c8e417023ba4443b41ada3b65/raw/67f89f4d9412aeecf46797ddc9829f52718f49d2/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/3bb15c8c8e417023ba4443b41ada3b65/raw/7e974e90246be88a927afc16c0cbd71b8ec2c3dd/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/3bb15c8c8e417023ba4443b41ada3b65/raw/238795e04c3a5c0474f720f1f72350c1403f7689/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/3bb15c8c8e417023ba4443b41ada3b65/raw/6978e94d2052536fe3631fabeb073a9e21cc09ac/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/3bb15c8c8e417023ba4443b41ada3b65/raw/8e91397b8cdd822c590499d10a04f44aff0f691f/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/3bb15c8c8e417023ba4443b41ada3b65/raw/eea70908baa00dfa51631fd97e477e920ea1ebb7/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W0](https://gist.githubusercontent.com/mellanox-github/3bb15c8c8e417023ba4443b41ada3b65/raw/158c76fb6cfab7194f9e0e22b35e32a48b262572/hpc-test-node-upstream_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W1](https://gist.githubusercontent.com/mellanox-github/3bb15c8c8e417023ba4443b41ada3b65/raw/ff553a77866e8013a87a1d8a3a1d50f3b2f4aede/hpc-test-node-upstream_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W2](https://gist.githubusercontent.com/mellanox-github/3bb15c8c8e417023ba4443b41ada3b65/raw/784701eb971f6c882c80f3ecf5efc8fed28c8e29/hpc-test-node-upstream_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W3](https://gist.githubusercontent.com/mellanox-github/3bb15c8c8e417023ba4443b41ada3b65/raw/c290afb17c2a8d3e75d984dd7013b96537e7a1da/hpc-test-node-upstream_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/3bb15c8c8e417023ba4443b41ada3b65/raw/481b8b333aa9467d16d5b1b115aa7f73337cd192/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/3bb15c8c8e417023ba4443b41ada3b65/raw/5939b7e0abbf2f3839e5cf175b21d7c731d3f9cf/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/3bb15c8c8e417023ba4443b41ada3b65/raw/970fbd293c0063dc6666e132ab55807dd86a6581/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/3bb15c8c8e417023ba4443b41ada3b65/raw/8ea17479779eab42d2f6fcd5569ed0d5d5c36554/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by yosefe** [comment]
@brminich there's a conflict now

**Comment by brminich** [comment]
I merged master to #3834 (I guess conflict comment was made to that PR)

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/11466/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 33 workers (click for details)</summary>

**Note:** the logs will be deleted after 30-Jul-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/89f91ff6af8d4ced2104de03e864d331/raw/fb7d5dc1f1012fa9d56d1a215a3209bc3c4091f4/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/89f91ff6af8d4ced2104de03e864d331/raw/fae917ca10827759a7c97bc5d421779f0e429b8a/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/89f91ff6af8d4ced2104de03e864d331/raw/8ffe9c9f46ad995a715c9832a9549d7540b6eb04/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/89f91ff6af8d4ced2104de03e864d331/raw/6a8849f095bcc996cad08c8dff761d8cf12716ba/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/89f91ff6af8d4ced2104de03e864d331/raw/372787308894c7fb5c6ada5906e1efbab0a85304/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/89f91ff6af8d4ced2104de03e864d331/raw/b78fe6ea39d3bf2214ea66dcb24a4cb488af63df/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/89f91ff6af8d4ced2104de03e864d331/raw/9ad3444ea92b2efd5040d24dcbb80270643e8cdf/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/89f91ff6af8d4ced2104de03e864d331/raw/4d80a2c67ee1eb1f2d95d2b9ac3338fc4bfe4c94/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/89f91ff6af8d4ced2104de03e864d331/raw/92498668351dab1fd0ef8b2e7b32f1bd1e7203bb/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W0](https://gist.githubusercontent.com/mellanox-github/89f91ff6af8d4ced2104de03e864d331/raw/afb4fead82d640e5189d8abfca8ffe9c909a9381/hpc-test-althca_W0) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W1](https://gist.githubusercontent.com/mellanox-github/89f91ff6af8d4ced2104de03e864d331/raw/d54d0f27ab522ecb5352d15b00c5d3b19c45c327/hpc-test-althca_W1) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W2](https://gist.githubusercontent.com/mellanox-github/89f91ff6af8d4ced2104de03e864d331/raw/761368f8b37505a46f1502d1df72bd4e2d1b8042/hpc-test-althca_W2) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W3](https://gist.githubusercontent.com/mellanox-github/89f91ff6af8d4ced2104de03e864d331/raw/fad97d00cf38623c3f83de3411573a09681528fd/hpc-test-althca_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/89f91ff6af8d4ced2104de03e864d331/raw/de43e09f5beee1761073dfbeb2ac8f2a13acbb81/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/89f91ff6af8d4ced2104de03e864d331/raw/3f539f3f942642c7601c0bb4ed3a578a2669042a/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/89f91ff6af8d4ced2104de03e864d331/raw/95a692bd6a2631c982bd2d3bd418341c1ccc375e/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/89f91ff6af8d4ced2104de03e864d331/raw/75e830be0cc9c01e9818c07064537da87c9d7dae/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/89f91ff6af8d4ced2104de03e864d331/raw/7fd7051204d5c296cde952f25edcffc7e09c71a9/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/89f91ff6af8d4ced2104de03e864d331/raw/425dde83914711afab8437866e054272224cd3ae/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/89f91ff6af8d4ced2104de03e864d331/raw/e4f6992727d568e9e7158883bf24c9a1268afebd/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/89f91ff6af8d4ced2104de03e864d331/raw/bc6f9c0a89799dbc5c77b22fd19db7e29bcdaa8d/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/89f91ff6af8d4ced2104de03e864d331/raw/25d771823a42fc718766905d3a3b9caf767fc9bc/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/89f91ff6af8d4ced2104de03e864d331/raw/4192c92c56f50bffb13e449e1ea08dcca24bdfc3/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/89f91ff6af8d4ced2104de03e864d331/raw/f60827451130c70ec6b17eace1d47d7cc704c625/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/89f91ff6af8d4ced2104de03e864d331/raw/f0cb36e28ad96cb61123601c668218cd11ef548e/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W0](https://gist.githubusercontent.com/mellanox-github/89f91ff6af8d4ced2104de03e864d331/raw/8977dee6adc0594cc72ac9c4b65241e71f0a8dfd/hpc-test-node-upstream_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W1](https://gist.githubusercontent.com/mellanox-github/89f91ff6af8d4ced2104de03e864d331/raw/87066c1f3cc57480073ef1318a8ce8d64eac8dcd/hpc-test-node-upstream_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W2](https://gist.githubusercontent.com/mellanox-github/89f91ff6af8d4ced2104de03e864d331/raw/55000a9cdd28beee9677f27724ae6a1098fddb17/hpc-test-node-upstream_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-upstream_W3](https://gist.githubusercontent.com/mellanox-github/89f91ff6af8d4ced2104de03e864d331/raw/8a9873bec9d42bf1154a8b410151181507c46258/hpc-test-node-upstream_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/89f91ff6af8d4ced2104de03e864d331/raw/e7a2b3dbd9625015e10477fae03bfa44fa165cab/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/89f91ff6af8d4ced2104de03e864d331/raw/9445de28feba763bd37048a3316330d119cabbb7/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/89f91ff6af8d4ced2104de03e864d331/raw/3a4ffe7033e427f7490e5e0e4c4a2114fdc923ba/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/89f91ff6af8d4ced2104de03e864d331/raw/d10f4e20a92356fee8ddc320a9abda73ef3a7e14/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Pull request merged**